tools/scripts/build/build-external-curricula-data-v2.ts

```diff
@@ -85,6 +82,21 @@ interface GeneratedBlock {
   meta: Record<string, unknown>;
 }
 
+// This enum is based on the `SuperBlockStage` enum in shared/config,
+// but with string value instead of number.
+enum SuperBlockStage {
+  Core = 'core',
+  English = 'english',
+  Professional = 'professional',
+  Extra = 'extra',
+  Legacy = 'legacy'
+}
```

Comment:
Maybe this is nitpicky, but do we need this enum? Why not just use strings as keys?